content/en/blog/_posts/2018-10-08-support-for-azure-vmss.md

```diff
@@ -333,6 +333,6 @@ For the acs-engine (the unmanaged variety) on Azure docs can be found here: [[9]
 
 13) https://github.com/Azure/acs-engine/tree/master/examples/kubernetes-msi-userassigned
 
-14) https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/azure/auth/azure_auth.go
+14) https://github.com/kubernetes/kubernetes/blob/feature-serverside-apply/pkg/cloudprovider/providers/azure/auth/azure_auth.go
```

Comment:
👍

Reply:
@Aresforchina . I checked the links on the whole page. The links are set up a bit differently than expected. I found that these links did _not_ work from within the text:
- link number 3
- link number 14

Can you check in the text to make sure that the links match what is in the link Reference section? 😃 

The Kubernetes website local links forward correctly in the text, but also could be linked up in the Reference section as well.

Reply:
@kbhawkey  OK!I have alreay fix them .Thanks for your suggestion.

Reply:
Check out line 314 and make sure it matches the link in line 68.

Reply:
Folks, please try avoid pointers to the master branch which is under constant change. References to source code in general should be avoided. If we do have a real need to do so, please use pointer to a specific tag in the git repo.

Reply:
Are you suggesting other changes? The links to `cloudprovider` seem susceptible to change.

Reply:
I was wondering something like although `feature-serverside-apply` might not be a good choice. It smells like a temporary working branch for a feature named "server-side apply". When that feature is merged, the branch may go away. I may be wrong if the `feature-serverside-apply` is the only branch to reference. If we have a choice, we should use tags like `1.16.0`.

Reply:
Right. I am not familiar with what is happening in the cloud provider area.

Reply:
@ I did not find the cloud provider like azure in tags of 1.16.0.  should I look for for it in other tags?

Reply:
Yup. That is exactly where the problem is. We may need more digging into the upstream code to understand the correct repo/tag to point to. @feiskyer can you help?

Reply:
it is https://github.com/kubernetes/kubernetes/tree/release-1.17/staging/src/k8s.io/legacy-cloud-providers/azure/auth

Reply:
Thanks for fixing those links

Reply:
@feiskyer Thanks for the help. That is a good URL (permanent and valid) to use.